### DISCUSSION TITLE
The running test code reports an error on the latest sahi version

### DISCUSSION THREAD

**Discussion by HuLu65** [obss/sahi#775] [General]
When I ‘import torchvision’ reported an error the segfault core was dumped
This is my test code：
![image](https://user-images.githubusercontent.com/82073527/206970647-25a1d73b-d25a-4b8f-bcdc-a7f887c34c25.png)
I guess The latest version of Sahi cannot be imported into torchvision

**Comment by fcakyon** [comment]
will look into that 👍🏻 what is your python, pytorch and torchvision versions?

**Reply by HuLu65** [reply]
pyth
![image](https://user-images.githubusercontent.com/82073527/206979087-ea567b15-ebac-4f8d-88d3-5acf98e545fa.png)
on 3.7.15

**Reply by HuLu65** [reply]
I lowered the version of sahi, but a new problem arises.
This is my test code:
![image](https://user-images.githubusercontent.com/82073527/206979335-7470b25c-dddf-46c8-8174-4a366b8e8b03.png)
![image](https://user-images.githubusercontent.com/82073527/206979435-7379b50a-bab2-4847-a52b-63d64da3fbf9.png)

**Reply by fcakyon** [reply]
At what sahi version are you getting these errors? Have you tried with v0.11.6?

**Reply by HuLu65** [reply]
sahi 0.10.5

**Comment by HuLu65** [comment]
sahi 0.10.6 is good

**Comment by HuLu65** [comment]
Thanks！I have a question whether yolov7 integration is considered
